src/core/Banner/__snapshots__/banner.test.tsx.snap

```diff
@@ -37,7 +37,6 @@ exports[`<Banner /> LivePreview story renders snapshot 1`] = `
     </div>
     <button
       class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-reo6um-MuiButtonBase-root-MuiIconButton-root"
-      data-testid="iconButton"
```

Comment:
are the test ids not used anymore or something? Thank you!

Reply:
Test-Id was added to the main component by mistake. I just removed it and added the Test-Id to the Test story! :D

Reply:
Ahh got it! Thanks so much for correcting it 😆